lago/plugins/vm.py

```diff
@@ -374,13 +374,21 @@ def extract_paths(self, paths, *args, **kwargs):
         return self.provider.extract_paths(paths, *args, **kwargs)
 
     def export_disks(
-        self, standalone=True, dst_dir=None, compress=False, *args, **kwargs
+        self,
```

Comment:
Not per this PR - but the docstring is still a big problem for us, as what you'll see in the SDK is "Thin method that just uses the provider". Guess we could write some wrapper to update the docstring from the service provider dynamically.